### DISCUSSION TITLE
How to change the port for hyperdiv docs?

### DISCUSSION THREAD

**Discussion by XinChenQC** [hyperdiv/hyperdiv#68] [Help] [answered]
Dear developers,

Thanks for providing this usage package. I have a question about the hyperdiv docs?
When I type:
`hyperdiv docs`
The website runs on 8988 port. Can I change the port? Because it's running on my server. I can open new ports for safety sake.

**Comment by mariusnita** [comment, marked as answer]
Yes you can do this by setting the `HD_PORT` environment variable to a different port. For example in bash-like shells:
```sh
HD_PORT=7777 hyperdiv docs
```

**Reply by XinChenQC** [reply]
Solved, thank you so much!
